Updating sd_write for MacOS compatibility and to make post-burn proce…

rshaltry opened the pull request:
…ss more flexible

thcp committed:
Docker compose install added.

thcp committed:
Merge pull request #10 from thcp/dev/docker-compose

Docker compose install added.

Commit:
Updating sd_write for MacOS compatibility and to make post-burn process more flexible

Commit:
Moving 'cgroup_enable=cpuset cgroup_enable=memory' to sd_write to update init settings before initial boot

thcp reviewed (approved):
Nice job! Thanks for your time !

thcp commented:
@rshaltry We need your signature :) https://help.github.com/articles/about-commit-signature-verification/

thcp committed:
Force installation of docker 18.06

Currently, only docker 18.06 is validated to work with Kubernetes, so to avoid future surprises, we are forcing the installation of v18.06

thcp committed:
Merge pull request #2 from thcp/dev/docker-version-update

Dev/docker version update

Commit:
Merge branch 'master' of https://github.com/Enginexp/pi_kube into Enginexp-master

Commit:
Merge branch 'Enginexp-master'

Commit:
Merge branch 'features/macos-compatibility' of https://github.com/rshaltry/pi_kube into rshaltry-features/macos-compatibility

Commit:
Merge branch 'rshaltry-features/macos-compatibility'

Commit:
Merge branch 'master' of https://github.com/rshaltry/pi_kube into merge-macos-updates
